wscript

```diff
@@ -84,6 +86,54 @@ def _setup_openmp(conf):
     else:
         conf.end_msg("OpenMP found.", 'GREEN')
 
+def _ok_color(var, val):
+    if var == val:
+        return "GREEN"
+    else:
+        return "YELLOW"
+
+def _warn_color(var, val):
+    if var == val:
+        return "YELLOW"
+    else:
+        return "GREEN"
+
+def _err_color(var, val):
+    if var == val:
+        return "RED"
+    else:
+        return "GREEN"
+
+def _get_git_details(conf):
+    global APPNAME, VERSION, COMMIT, BRANCH
+
+    results = subprocess.run('git describe --dirty', shell=True, check=False, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Then they/we should make sure it gets build from a git dir. Perhaps you are suggesting that we explicitly enforce git as a dependency, rather than it being a fact from the code in this PR (which it is not) - the two are very different things.

I don't believe we should be making git a dependency, I'm not even sure how that works or, semantically, what it even means.